### PR TITLE
Test email via regex

### PR DESCRIPTION
**Pull request by Nainterceptor**
Checklist

- [x] I've added a mention in CHANGELOG.md

### CODE REVIEW

**Comment by Nainterceptor** [comment]
[Injection email](https://trello.com/c/kn0I8v5X/344-injection-email)

File: models/user.mjs [hunk 1]
```diff
@@ -55,6 +55,11 @@ const { Schema } = mongoose;
 const UserSchema = new Schema({
   email: {
     type: String,
+    // Same regex than html5 validator, close to RFC compliance but not strictly equivalent.
+    match: new RegExp(
+      '^[a-zA-Z0-9.!#$%&\'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}'
```
Comment (Scttpr) [inline models/user.mjs]: I'm not sure this regex limits number of char before `@` (it's set to be between 1 and x) should it be the case ?

File: services/mail.mjs [hunk 2]
```diff
@@ -12,7 +12,8 @@ const { fileURLToPath } = nodeUrl;
 const currentPath = dirname(fileURLToPath(import.meta.url));
 const DEFAULT_LANG = 'fr';
 
-export default async function sendMail(to, options = {}) {
+export default async function sendMail(recipients, options = {}) {
```
Comment (Scttpr) [inline services/mail.mjs]: Was `to` already an array ? If not I'm surprised that there are no other changes in this function.

File: models/user.mjs (hunk 1 above)
Comment (Nainterceptor) [reply models/user.mjs]: there is not limit in [RFC 5322](https://tools.ietf.org/html/rfc5322) on email addresses, only on the line that should be less than 78 chars and must be less than 1k chars.

**Pull request merged**